Asterix/optics/phase_amplitude_functions.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from Asterix.optics.propagation_functions import mft
 from Asterix.utils import crop_or_pad_image, rebin
+from Asterix.utils.save_and_read import quickfits
```

Comment:
can be removed, quickfits only used for debugging

Reply:
True. How come flake8 didn't catch an unused import?

Reply:
Ah. This is why.
https://github.com/johanmazoyer/Asterix/blob/fe428422d1627e83b19bf5950d746bc682725df8/.flake8#L26

Reply:
And this is why:
https://github.com/johanmazoyer/Asterix/pull/76#discussion_r990008453